File: PaMVPDemo/lib-core/src/main/java/com/pa/core/config/GlobalConfigManager.java
package com.pa.core.config;

import com.pa.core.imagehelper.GlideManger;
import com.pa.core.nethelper.RetrofitManger;

public class GlobalConfigManager {

    public static final void initNet(NetworkConfig networkConfig){
        RetrofitManger.getInstance().init(networkConfig);
    }

    public static final void initImageLoader(ImageLoaderConfig imageLoaderConfig){
        GlideManger.getInstance().init(imageLoaderConfig);
    }
}

File: PaMVPKotlinDemo/lib-core/src/main/java/com/pa/core/config/GlobalConfigManager.kt
package com.pa.core.config

import com.pa.core.imagehelper.GlideManger
import com.pa.core.nethelper.RetrofitManger

object GlobalConfigManager {

    fun initNet(networkConfig: NetworkConfig) {
        RetrofitManger.getInstance()?.init(networkConfig)
    }

    fun initImageLoader(imageLoaderConfig: ImageLoaderConfig) {
        GlideManger.getInstance()?.init(imageLoaderConfig)
    }
}
